libcxx/src/atomic.cpp

```diff
@@ -101,6 +105,24 @@ static void __libcpp_platform_wake_by_address(__cxx_atomic_contention_t const vo
   _umtx_op(const_cast<__cxx_atomic_contention_t*>(__ptr), UMTX_OP_WAKE, __notify_one ? 1 : INT_MAX, nullptr, nullptr);
 }
 
+#elif defined(__Fuchsia__)
+
+static inline zx_futex_t const* __libcpp_zx_futex(__cxx_atomic_contention_t const volatile* ptr) {
```

Comment:
```suggestion
static inline zx_futex_t const* __get_zx_futex(__cxx_atomic_contention_t const volatile* ptr) {
```

Let's move away from `__libcpp_` prefixes everywhere. I know it's confusing because we have a lot of them, but they shouldn't exist in most cases.